### PR TITLE
Add flake8 and isort linting and lint files

### LINKED ISSUE #7: Coding style
Do we want to specify some sort of codings style? [PEP8](https://pep8.org/) seems like a verry reasonable starting-point to me. Using flake8 this could easily become part of a CI.

### PR DESCRIPTION
**Pull request by e1mo**
This adds PEP8 checking as discussed in #7. This also provides a github
action workflow for automatic tests.

Further, I fixed up most offending files, excluding the admin.py and
models.py due the pending #5 and #8.

Close #7

### CODE REVIEW

File: src/setup.cfg [hunk 1]
```diff
@@ -17,6 +17,7 @@ classifiers =
     Programming Language :: Python :: 3.6
     Programming Language :: Python :: 3.7
     Programming Language :: Python :: 3.8
+    Programming Language :: Python :: 3.9
```
Comment (adnidor) [inline src/setup.cfg:20]: I'm not sure, but I think this should go in a separate commit

**Comment by e1mo** [comment]
Yeah for sure, there are 2-3 other things in the setup.cfg that need a n
update, I'll prepare a PR later today.

File: .github/workflows/style.yml [hunk 2]
```diff
@@ -0,0 +1,53 @@
+---
+name: Code Style
+on:
+  - push
+  - pull_request
+jobs:
+  isort:
```
Comment (e1mo) [inline .github/workflows/style.yml:7]: We have the issue of this workflow running twice when PRs are created from this repo. This if clause will avoid this running twice by not running the workflow if the PR is from the same repo.

```suggestion
  isort:
     if: github.event_name == 'push' || github.event.pull_request.head.repo.full_name != github.repository
```

Does anyone have a better idea? Maybe @margau ?

Sources:

- <https://github.community/t/duplicate-checks-on-push-and-pull-request-simultaneous-event/18012/5>
- <https://github.com/corejam/corejam/pull/7>

File: src/chaosinventory/base/admin.py [hunk 3]
```diff
@@ -1,3 +1,5 @@
 from django.contrib import admin
 
+# TODO: Remove noqa once admin is implemented
```
Comment (adnidor) [inline src/chaosinventory/base/admin.py:3]: This comment seems to be outdated

**Review by n0emis — APPROVED** [review]

File: .github/workflows/style.yml (hunk 2 above)
Comment (e1mo) [reply .github/workflows/style.yml:7]: Added in 60cfee9

**Review by clerie — APPROVED** [review]

**Review by adnidor — APPROVED** [review]
